CONTRIBUTING.md

```diff
@@ -0,0 +1,202 @@
+To run pmm-managed with a new changes just run `make env TARGET=run` to update `pmm-managed` running in container.
+
+### PMM Client
+
+* Clone [pmm-agent repository](https://github.com/percona/pmm-agent) and [pmm-admin repository](https://github.com/percona/pmm-admin).
+* Navigate to the root folder of pmm-agent. 
+* Run `make setup-dev` to connect pmm-agent to PMM Server.
+  * This command will register local pmm-agent to PMM Server and generate config file `pmm-agent-dev.yaml`
+  * Once it's connected just use `make run` to run pmm-agent.
+* To work correctly pmm-agent needs exporters installed on the system.
+  * All pathes to exporters binaries is configured in `pmm-agent-dev.yaml`, so they can be changed manually
+  * Exporters can be setup by building each of them or by downloading pmm-client tarball from [percona.com](https://www.percona.com/downloads/pmm2/) and copying binaries to exporters_base directory configured in a `pmm-agent-dev.yaml` file.
+
+### Exporters
+
+Exporters by itself is an independent applications, so each of them contains their own README files explaining how to contribute [see PMM Client](#PMM-Client).
+
+### UI
+
+See [Grafana Dashboards Contribution Guide](https://github.com/percona/grafana-dashboards/blob/main/CONTRIBUTING.md).
+
+< TODO:Explain how to contribute to Grafana >
+
+## Tests
+    
+In a PMM we have 4 kind of tests.
+    
+### Unit tests
+    
+The first one is a Unit testing, so we have unit tests in each repository mentioned above. each of repositories has it's own instruction how to run unit tests.
+    
+### API tests
+    
+API tests are included into pmm-managed repository and located in [api-tests directory](https://github.com/percona/pmm-managed/tree/main/api-tests). API tests runs against running PMM Server container.
+    
+### End to End (E2E) tests
+    
+End to End tests are located in [pmm-qa repository](https://github.com/percona/pmm-qa). They includes UI tests and CLI tests.
+Please see [readme](https://github.com/percona/pmm-qa#readme) for details on how to run theese.
+
+## Submitting a Pull Request
+
+See [Working with Git and GitHub](GIT_AND_GITHUB.md)
+
+As a PR created you are responsible to:
+* make sure PR is ready (linted, tested and etc)
+* make sure it is reviewed (ask for review, ping reviewers and etc)
+* make sure it is merged
+  * merge when it is reviewed and tested
+  * ask code owners/admins to merge it if merging is blocked for some reason
+
+### When can the code be merged?
+
+* code can only be merged via PRs
+* at least 2 reviews are assigned and they should have approved the PR
+* tests and checks passed
+* FB was tested by QA (if it applies)
+
+If above criteria are met - anyone (PR owner or any other developer) can merge the code.
+
+In case of CODEOWNERs they would merge it.
+
+If there are any issues with tests, checks or CODEOWNER is absent but there are 2 reviews - PR owner should ask anyone from PMM Admins to merge PRs.
```

Comment:
we have a separate block for merging rules in GIT_AND_GITHUB.md, should we duplicate it here?